airgun/widgets.py

```diff
@@ -699,7 +699,8 @@ class Search(Widget):
 
     ROOT = (
         './/div[contains(@class, "toolbar-pf-filter") or contains(@class, "title_filter")'
-        'or contains(@class, "dataTables_filter") or @id="search-bar"]'
+        'or contains(@class, "dataTables_filter") or @id="search-bar" '
+        'or @data-ouia-component-id="table-toolbar"]'
```

Comment:
@lhellebr why not fix this in the view, rather updating the the common locators? 

We get into a vicious cycle where we fix the common locator (for some entity) and then break more tests. then again update the common locator fix for multiple tests.

I would recommend you fix this in the specific entity view rather than patching the common locator.

@mstrlc has that data in detail.

Reply:
This is more generic. I found at least one another page where this widget with this id is used, i.e. `https://<sat_fqdn>/models` , and it's defined in the main foreman project in `webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/TableIndexPage.js`. It can (and perhaps will) be used in more places. I think this should remain generic in airgun as well.

Reply:
This looks like the PF4 search box and for that reason, we already have a separate class. Once you start updating the PF3 search box with PF4 locators create the mess. https://github.com/SatelliteQE/airgun/pull/906 already have the PF4 search done. you can wait or update along with his PR for PF4  
![image](https://github.com/SatelliteQE/airgun/assets/3190629/45586d48-0685-4ef4-8322-ff2dc833f407)

Reply:
That also confused me, yes. For some reason, the old Search widget is used in airgun for Webhooks, I can't say I understand why. And with the fix, it works. I'll try to look at it once again when I change it to PF4 but there were some issues for Webhooks.

Reply:
Okay, let me know what the findings you have. @mstrlc  also working on similar concept and raised PR here https://github.com/SatelliteQE/airgun/pull/913/files might be that help.

Reply:
Thanks, @omkarkhatavkar , the last linked PR is exactly what I needed. Closing this PR and opening another that just switches Webhooks to PF4Search - it will work once Matyas's PR is merged.